packages/wasm-bindings/src/lib.rs

```diff
@@ -7,6 +7,82 @@ use wasm_bindgen::prelude::*;
 #[global_allocator]
 static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
 
```

Comment:
We should break up these functions into separate files based on what panel they are a part of. *Window* is the name of the window shell "panel". It would also be helpful if we can make the JS side aware of the "namespacing" of these functions to avoid collisions and make it clear when code is calling the window, the viewport panel, the properties panel, the layers panel, the graph panel, etc. One way to do this is by prefixing everything, however a more proper way of "namespacing" would be preferable.